app/controllers/application_controller.rb

```diff
@@ -35,7 +35,7 @@ def default_serializer_options
   end
 
   def ensure_http_referer_is_set
-    request.env['HTTP_REFERER'] ||= '/'
+    request.env["HTTP_REFERER"] ||= "/"
```

Comment:
Yes, this was intentional because we thought to clean up the obvious things rubocop suggested. We didn't think about it being confusing.